package bitmentor.api.exceptions

class UserNotFoundException: Exception()
